pkg/broker/ttl.go

```diff
@@ -23,26 +23,26 @@ import (
 )
 
 const (
-	// v02TTLAttribute is the name of the CloudEvents 0.2 extension attribute used to store the
+	// v03TTLAttribute is the name of the CloudEvents 0.2 extension attribute used to store the
```

Comment:
Hi @anishj0shi, In the comment, please change the **CloudEvents 0.2** to **CloudEvents 0.3**?

Reply:
good catch. I'll merge so we get this in, and I'll send a followup PR just for this small change.